### ISSUE TITLE
If Invite Collaborators option is present, interactive clickable lyrics disappear and vice-versa.

### ISSUE THREAD

**Issue by twistedmacaroni** [SpotX-Official/SpotX#42]
## ℹ Computer information

- Spotify version: 1.1.85.895.g2a71e1b8
- Windows version: 11
- PowerShell version: 5.1.22000.588

## 📝 Description of the problem

1. When I fresh installed Spotify, I used SpotX command without logging it in. That resulted in me getting interactive lyrics on my account (they are available for only a few accounts ATM).
2. When I re-installed the script (ran Uninstall.bat, opened Spotify (with ads), closed Spotify, installed the script again), the interactive lyrics disappeared and switched to the normal time-synced one, but I got the Invite Collaborators icon next to the Enhance button under my playlist.
3. Is it possible to have both the options simultaneously? Thanks

## 📷 Screenshots

<details>
<summary><small>Screenshot</small></summary><p>
No Invite Collaborators option present but the interactive lyrics are there:

![image](https://user-images.githubusercontent.com/93189292/168559103-38ce0941-e07d-4ba0-aeb4-f18c05b41c91.png)

![image](https://user-images.githubusercontent.com/93189292/168560539-2cd9ba51-fcb3-489b-b120-8e1a887784a3.png)

Invite Collaborators option appears after a uninstall/install of SpotX but the interactive lyrics are gone:

![image](https://user-images.githubusercontent.com/93189292/168559366-6fa0e1e7-3b03-4f1c-ab0b-a8d08bf4e19f.png)

![image](https://user-images.githubusercontent.com/93189292/168559746-ef135726-8b24-4c6d-afaa-bc67ea77e5e7.png)

</small></summary><p>

</details>

**Comment by amd64fox** [comment]
it's not a bug, I didn't turn on this button on purpose, it's too ugly (
I can activate the collaboration function in the playlist menu.
It will look like this

![2022-05-16 152104](https://user-images.githubusercontent.com/62529699/168591792-acf6c4f6-e591-49ca-9bb5-28c945d99188.jpg)

will that be enough ?

**Comment by twistedmacaroni** [comment]
sure but can't you just tell me a way to enable it manually? the context menu would be great too but if there's a chance to enable it manually... thanks anyway

**Comment by amd64fox** [comment]
manual
- Open folder `AppData\Roaming\Spotify\Apps`
- Open the `xpui.spa` file, for example using the `7zip archiver`

`xpui.js`
- Open the `xpui.js` file for editing and search for the line `Enable Playlist Permissions flows for Prod",default:!1`
- Change at the end of the found text this `default:!1` to this `default:!0`
- save file and close

`xpui.css`
- Open the `xpui.css` file for editing and find and delete text `.X1lXSiVj0pzhQCUo_72A{display:none}`
- save file and close

**Comment by twistedmacaroni** [comment]
Thanks! The button and the context menu option is available now so I guess you don't have to activate it?

**Comment by amd64fox** [comment]
I added an item to the playlist context menu as shown in the screenshot above, maybe it will be useful for some users.

A bug was also fixed if, when installing my patch, there was already a Spotify client, for some accounts the settings of experimental chips could go astray.

Also updated the post above to manually enable the button. Now you additionally need to edit the `xpui.css` file, it is only necessary for the icon to be visible.
